frontend/src/components/drawer.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import styled from '@emotion/styled';
+import MailIcon from '@mui/icons-material/Mail';
+import InboxIcon from '@mui/icons-material/MoveToInbox';
+import Divider from '@mui/material/Divider';
+import MuiDrawer from '@mui/material/Drawer';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Toolbar from '@mui/material/Toolbar';
+
+const StyledDrawer = styled(MuiDrawer)({
+  flexShrink: 0,
+  whiteSpace: 'nowrap',
+  boxSizing: 'border-box',
+  anchor: 'left',
+}, ({ expand }: StyledDrawerProps)=> ({
+  width: expand ? 60 : 200,
+  '& .MuiDrawer-paper': { width: expand ? 60 : 200 },
+}));
+
+interface StyledDrawerProps {
+  expand : boolean,
+}
+
+export default function Drawer({ expand } : StyledDrawerProps) {
+  return (
+    <StyledDrawer variant="permanent" expand={expand}>
+    <Toolbar />
+    <List>
+      {['홈', '탐색', '구독', 'originals'].map((text, index) => (
+        <ListItem button key={text}>
+          <ListItemIcon>
+            {index % 2 === 0 ? <InboxIcon /> : <MailIcon />}
+          </ListItemIcon>
+          <ListItemText primary={text} />
+        </ListItem>
+      ))}
+    </List>
+    <Divider />
+    <List>
+      {['홈', '탐색', '구독', 'originals'].map((text, index) => (
```

Comment:
```suggestion
      {['Home', 'Exploration', 'Subscriptions', 'whatever'].map((text, index) => (
```

![81F90CFE-912D-4FE1-9569-0E2DBFB789BF](https://user-images.githubusercontent.com/36983960/153740063-b21920b2-2155-4f72-aef4-77b8950f1078.png)
물론 샘플로 임의로 넣어두신거겠지만 다른것으로 확장할 수 있다는 의미로 다른 문자들이 들어가 있으면 좋을 것 같습니다 ㅎㅎ